### ISSUE TITLE
[Mac app] Horizontal rules invisible in exported PDF

### ISSUE THREAD

**Issue by tylerlong** [tylingsoft/markdown-plus#11]
![screen shot 2015-04-03 at 7 25 11 am](https://cloud.githubusercontent.com/assets/733544/6975742/9cc90d90-d9d2-11e4-9497-0a3edec3c211.png)

**Comment by tylerlong** [comment]
investigating...

**Comment by tylerlong** [comment]
Here is a another interesting test:

Open this page: https://gist.github.com/tylerlong/556196d7a5dba488c138  

Press `Cmd - P` to invoke print preview, You can see that the horizontal rules are invisible too!

So this might be an universal issue which has something to do with printing on Mac.

![screen shot 2015-04-03 at 7 43 21 am](https://cloud.githubusercontent.com/assets/733544/6975910/223a69c2-d9d5-11e4-9b03-ede4f53f37d9.png)

![screen shot 2015-04-03 at 7 43 59 am](https://cloud.githubusercontent.com/assets/733544/6975922/38cc0f1a-d9d5-11e4-9921-ece01c4965fc.png)

**Comment by tylerlong** [comment]
http://www.w3schools.com/tags/tag_hr.asp

```
In HTML5, the <hr> tag defines a thematic break.

In HTML 4.01, the <hr> tag represents a horizontal rule.

However, the <hr> tag may still be displayed as a horizontal rule in visual browsers, but is now defined in semantic terms, rather than presentational terms.
```

**Comment by tylerlong** [comment]
http://stackoverflow.com/questions/9140585/hr-not-seen-when-printing

> The background not printing is a preference setting in your browser. It's off by default to save ink. But if the hr (or any construct) is white and is showing on the coloured background on your screen, you might want to switch this setting on.

**Comment by tylerlong** [comment]
Issue fixed. Please wait for the next release of our Mac app. :beer:

Issues has been fixed in App version v1.3.6
